### ISSUE TITLE
Restore a perceivable label to Bookmark checkbox

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#862]
Part of #820

Now that we're using the bookmark icon `svg` in place of the text label, the checkbox is effectively unlabelled for a screen reader. 

Instead of replacing the content of `<span>Bookmark</span>` with the `svg` do something like this:

```
<label>
   <input>
   <span><svg></span>
   <span class="sr-only">Bookmark</span>
</label>
```

- Retain the original text label but make it visible to SR-only when we want the icon alone
- Add the icon before the `sr-only` span
